src/components/search/facets/custom-facets.vue

```diff
@@ -11,17 +11,17 @@
         </template>
         <template #default>
           <AllFilter v-slot="{ isSelected }" :facet="facet">
-            <CheckboxCardSelectedIcon v-if="isSelected" class="x-icon--l" />
-            <CheckboxCardUnselectedIcon v-else class="x-icon--l" />
+            <CheckboxCardSelectedIcon v-if="isSelected" class="x-icon-lg" />
```

Comment:
After updating `x-components` you will need to change these components by:

CheckboxSelectedIcon
CheckboxUnselectedIcon